docs/documentation/stories/moving-into-the-cli.md

```diff
@@ -0,0 +1,63 @@
+# Moving your project to Angular-CLI
+
+It's easy to move your existing project to use Angular-CLI.
+The most straightforward way is to to make a new CLI project and copy over your app files.
+
+Let's see how to do it step by step.
+Here we use a project made with the [official QuickStart](https://github.com/angular/quickstart)
+as an example, but you should be able to adjust these instructions to other setups.
+
```

Comment:
Can you clarify how this edit would help? If an app is using the quickstart setup, these are the instructions they should follow and would need no adjustment. You only need to adjust if you're not using the quickstart boilerplate.